src/EnergyPlus/OutputProcessor.hh

```diff
@@ -74,6 +74,51 @@ struct EnergyPlusData;
 
 namespace OutputProcessor {
 
+    enum class eTimeStepType : int
+    {
+        Zone = 0,
+        HeatBalance = 1,
+        HVAC = 2,
+        System = 3,
+        Plant = 4
+    };
+    std::array<std::string_view, 5> constexpr sTimeStepType = {"Zone", "HeatBalance", "HVAC", "System", "Plant"};
+    inline eTimeStepType getTimeStepTypeEnum(std::string const &s)
```

Comment:
I _think_ this function might not be used anymore, it was just a temporary function while transitioning the arguments.

Reply:
Indeed it seems it is not used anymore.  Removing in a follow-up commit.